packages/components/card/src/Card.tsx

```diff
@@ -37,8 +38,21 @@ export default defineComponent({
       return children.value.some(node => node.type && (node.type as any).name === 'IxCardGrid')
     })
 
+    const [selected, setSelected] = useControlledProp(props, 'selected', () => false)
```

Comment:
```suggestion
    const [selected, setSelected] = useControlledProp(props, 'selected', false)
```

简单类型的初始值直接给就好了，引用类型的值，才需要用工厂函数。

Reply:
fixed